src/styles.scss

```diff
@@ -106,6 +106,12 @@ pre[class*='language-'] {
   border-radius: 0 !important;
 }
 
+.table-footer-highlight {
+  background-color: #03ada5;
+  padding: 0px !important;
+  text-align: center !important;
```

Comment:
i would align it, like it is in the column (left)
and i would not set that background color, looks a bit out of place.
Again, @lab900-senn-lambrecht what do you think

Reply:
This is just example styling, doesn't matter that much I guess.

Reply:
yes it was just to show what you can do